tests/ga/test_multi_config_extension.py

```diff
@@ -1129,12 +1129,22 @@ def __assert_invalid_status_scenario(self, protocol, fail_code, mc_status="NotRe
         sc_dependent_handler = self._assert_and_get_handler_status(aggregate_status=protocol.aggregate_status,
                                                                    handler_name="Microsoft.Powershell.ExampleExtension",
                                                                    status="NotReady", message=err_msg)
-        self.assertTrue(all('runtimeSettingsStatus' not in handler for handler in sc_dependent_handler))
-
-        sc_independent_handler = self._assert_and_get_handler_status(
-            aggregate_status=protocol.aggregate_status, handler_name="Microsoft.Azure.Geneva.GenevaMonitoring",
-            handler_version="1.1.0", status="NotReady", message=err_msg)
-        self.assertTrue(all('runtimeSettingsStatus' not in handler for handler in sc_independent_handler))
+        for handler in sc_dependent_handler:
+            extension_status_msg = get_keypath_or_throw(handler, "runtimeSettingsStatus/settingsStatus/status/formattedMessage/message",
+                msg="SC extension '{0}' did not report extension status when its dependency failed.".format(handler['handlerName']))
+            
+            self.assertEqual(err_msg, extension_status_msg, "Expected the dependent extension's status to match {0}, but got '{1}' instead.")
+
+        sc_independent_handler = self._assert_and_get_handler_status(aggregate_status=protocol.aggregate_status,
+                                                                    handler_name="Microsoft.Azure.Geneva.GenevaMonitoring",
+                                                                    handler_version="1.1.0", status="NotReady", message=err_msg)
+        for handler in sc_independent_handler:
+            extension_status_msg = get_keypath_or_throw(handler, "runtimeSettingsStatus/settingsStatus/status/formattedMessage/message",
+                msg="SC extension '{0}' did not report extension status even though it is independent.".format(handler['handlerName']))
+            
+            # TODO: This might be a bug; independent extensions probably should not fail when 
+            # other extensions do, but that's what is happening here.
+            self.assertEqual(err_msg, extension_status_msg, "Expected the independent extension's status to match {0}, but got '{1}' instead.")
```

Comment:
The exisiting behavior of this test treated dependent SC extensions and independent SC extensions the same, but that doesn't seem right. Left this `TODO` here and opened a bug. When that (potential) bug is fixed, this test will fail and should be updated to reflect `sc_independent_handler` succeeding.